docs/install.md

```diff
@@ -384,14 +384,12 @@ This step is optional but makes it much easier to connect to and administer the
 Host app
   Hostname fsrrszf5qw7z2kjh.onion
   User <ssh_user>
-  ProxyCommand connect -R remote -5 -S localhost:9050 %h %p
 Host mon
   Hostname yt4j52ajfvbmvtc7.onion
   User <ssh_user>
-  ProxyCommand connect -R remote -5 -S localhost:9050 %h %p
 ```
 
-Now you can simply use `ssh app` and `ssh mon` to connect to each server, and it will be stored in the Tails Dotfiles persistence. You might see the message "Enter SOCKS5 password for amnesia@localhost" — this can be ignored, just always press Enter.
+Now you can simply use `ssh app` and `ssh mon` to connect to each server, and it will be stored in the Tails Dotfiles persistence.
```

Comment:
Actually not sure. @dolanjs says that everybody gets the prompt, and he definitely does, which is why I retracted my statement. But I'm not getting it. We should maybe do some checking to see whether this is consistent with the latest version of Tails.
![amnesia_ssh](https://cloud.githubusercontent.com/assets/1161532/6854375/21bea59e-d3ae-11e4-934b-7143c8f79b71.png)